MAP/src/utils/constant.js

```diff
@@ -0,0 +1,15 @@
+
+export const mapStyleConstant = {
+    NO_LOCK_DOWN: "NO LOCKDOWN",
```

Comment:
these shouldn't be here, because they're DSL 1 specific. (are they even used?)

Reply:
okay removed

Reply:
only 1 was removed, but the whole block is questionable

Reply:
okay I got removed, I was planning to use it for spanhoot